CONTRIBUTING.md

```diff
@@ -58,4 +59,6 @@ Please keep the following in mind:
 
 - Please do not commit any configuration files or directories inside the exercise other than pubspec.lock and pubspec.yaml
 
-- Be sure to add it to the appropriate place in the `config.json` file. Also, please run `bin/fetch-configlet && bin/configlet lint .` to ensure the exercise is configured correctly.
+- Be sure to add the new exercise to the appropriate place in the `config.json` file.
+
+- Please run `bin/fetch-configlet && bin/configlet lint .` to ensure the exercise is configured correctly.
```

Comment:
Do we not have to specify track-id to configlet in this case?

Reply:
Good catch